components/transactions/TransactionItem.js

```diff
@@ -52,7 +53,7 @@ const getDisplayedAmount = (transaction, collective) => {
     // Expense Debits should display the Amount with Payment Method fees only on collective's profile
     return isSelf ? transaction.netAmount : transaction.amount;
   } else if (transaction.isRefunded) {
-    if ((isSelf && !transaction.isRefund) || (transaction.isRefund && isCredit)) {
+    if (hasExpense || (isSelf && !transaction.isRefund) || (transaction.isRefund && isCredit)) {
```

Comment:
:+1:

Reply:
Looking at this again, I realize better to use `isExpense` and check for `transaction.kind` instead since it's a more direct comparison and also we are making things consistent. I've just changed that.